pressbooks/pressbooks
# Improve styling of top nav

Fix for https://github.com/pressbooks/pressbooks/issues/3152#issuecomment-1447210560

To test:
1. Check to see if the PB logo appears in top nav bar for all logged in users. Link should point to pressbooks.com. Screen reader text should read 'About Pressbooks'
2. Check to see if profile avatar receives thin white border upon hover, focus, and active states
3. Check to see if profile avatar has more right padding

![Screenshot from 2023-02-27 17-33-33](https://user-images.githubusercontent.com/13485451/221729727-52b7509d-40c2-414b-b4c1-a37d552fb342.png)
